packages/web-api-scan-runner/docker-image-config/Dockerfile

```diff
@@ -1,37 +1,24 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-FROM node:12-slim
-
-# Install latest chrome dev package and fonts to support major charsets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
-# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
-# installs, work.
-RUN apt-get update \
-    && apt-get install -y wget gnupg \
-    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
-    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
-    && apt-get update \
-    && apt-get install -y google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 libxtst6 procps \
-    --no-install-recommends \
-    && rm -rf /var/lib/apt/lists/*
+FROM mcr.microsoft.com/windows:1809-amd64
 
-# Install app dependencies
-COPY package.json ./
-
-RUN npm install \
-    # Force npm package resolution
-    && npx npm-force-resolutions \
-    && npm install \
-    # Add user so we don't need --no-sandbox.
-    # same layer as npm install to keep re-chowned files from using up several hundred MBs more space
-    && groupadd -r pptruser && useradd -r -g pptruser -G audio,video pptruser \
-    && mkdir -p /home/pptruser/Downloads \
-    && chown -R pptruser:pptruser /home/pptruser \
-    && chown -R pptruser:pptruser /node_modules
-
-# Run everything after as non-privileged user.
-USER pptruser
+ENV NODE_VERSION 12.20.2
+
+USER ContainerAdministrator
```

Comment:
can we switch to non-admin after setup done?

Reply:
we need the Container admin to run the poweshell script and create the ip static route inside the container, otherwise the script fails to run

Reply:
What I mean after script switch to `USER ContainerUser`

Reply:
same